src/App.js

```diff
@@ -18,7 +19,11 @@ function App() {
           <Route path="/games" element={<Games />} />
           <Route path="/Login" element={<Login />} />
           <Route path="/games/spaceinvaders" element={<SpaceInvadersGame />} />
+//not sure what it was supposed to be here merge pull by augusto - reviewed by jack
```

Comment:
@aumfreire 
I do not know what is supposed to be here
If you can explain me that would be incredible please :)